src/praisonai/praisonai/bots/discord.py

```
@@ -496,3 +520,97 @@ def _convert_message(self, message) -> BotMessage:
+        
+        try:
+            if chat_id.isdigit():
+                # DM to user
+                user = await self._client.fetch_user(int(chat_id))
+                await user.send(text)
+            else:
+                # Channel message
+                channel = self._client.get_channel(int(chat_id))
+                if channel:
+                    await channel.send(text)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)